controllers/provisioners/eks/scaling/launchtemplate.go

```diff
@@ -311,6 +349,27 @@ func (lt *LaunchTemplate) blockDeviceList(volumes []v1alpha1.NodeVolume) []*ec2.
 	return sortTemplateDevices(devices)
 }
 
+func launchTemplateLicenseSpeficicationRequest(s []string) []*ec2.LaunchTemplateLicenseConfigurationRequest {
+	var output []*ec2.LaunchTemplateLicenseConfigurationRequest
+	for _, v := range s {
+		output = append(output, &ec2.LaunchTemplateLicenseConfigurationRequest{
+			LicenseConfigurationArn: aws.String(v),
+		})
+	}
+	return output
+}
+
+func launchTemplatePlacementRequest(input *LaunchTemplatePlacementInput) *ec2.LaunchTemplatePlacementRequest {
+	if input == nil {
+		return &ec2.LaunchTemplatePlacementRequest{}
+	}
+	return &ec2.LaunchTemplatePlacementRequest{
+		AvailabilityZone:     aws.String(input.AvailabilityZone),
```

Comment:
Since this is an optional field - it might be problematic to set it.
We should only set `AvailabilityZone` if it is not empty.